update-packages.sh

```diff
@@ -10,6 +10,16 @@ log() {
   echo "$(date -u +"%Y-%m-%d %H:%M:%S UTC") [${1}] - ${2}"
 }
 
+apt_get_update() {
```

Comment:
Is there a reason why this function is in the update script again?

Reply:
probably forgot to remove it again after debugging the packages that didn't get updated correctly.